.clang-format

```diff
@@ -0,0 +1,104 @@
+---
```

Comment:
Kinda funny how clang-format doesn't format .clang-format files :>

Reply:
`.clang-format` is a YAML file. Could use Prettier for that. Let me know the kind of format you want and I can fix it.

Reply:
Haha I was just making a joke, sadly it's not possible to leave informational comments on github :)